src/numJewelsInStones.js

```diff
@@ -28,7 +28,13 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let coinsidenceCount = 0;
+  for (let i = 0; i < stones.length; i++) {
+    if (jewels.includes(stones[i]) && stones[i].match(/[a-z]/gi)) {
+      coinsidenceCount++;
+    }
+  }
+  return coinsidenceCount;
```

Comment:
new line before return
```suggestion

  return coinsidenceCount;
```

Reply:
okey! Thanks.